src/main/java/net/mcreator/ui/SplashScreen.java

```diff
@@ -59,19 +60,18 @@ public SplashScreen() {
 		logo.setBounds(24 + 8 - 4, 70, 350, 63);
 		imagePanel.add(logo);
 
-		JLabel version = new JLabel(
-				"VERSION " + (Launcher.version != null ? Launcher.version.getMajorString() : "1234.5"));
+		JLabel version = new JLabel("VERSION " + (Launcher.version != null ? Launcher.version.getMajorString() : "1234.5"));
 		version.setFont(splashFont.deriveFont(18f));
 		version.setForeground(Color.white);
 		version.setBounds(30 + 10 - 4, 129, 500, 45);
 		imagePanel.add(version);
 
 		if (Launcher.version != null && Launcher.version.isSnapshot()) {
-			JLabel snpashot = new JLabel("Snapshot - not for production use!");
-			snpashot.setFont(splashFont.deriveFont(14f));
-			snpashot.setForeground(new Color(255, 92, 82));
-			snpashot.setBounds(30 + 10 - 4, 165, 500, 45);
-			imagePanel.add(snpashot);
+			JLabel snapshot = new JLabel("Snapshot - not for production use!");

```

Comment:
This is not translated, but should be

Reply:
:x: Won't fix
L10N is loaded while SplashScreen is shown, calling L10N from here causes app crash with exceptions from L10N.
Thus all 3 strings in the SplashScreen (`splash_screen.version`, `splash_screen.snapshot` and `splash_screen.note`) become untranslatable as they are requested **BEFORE** showing it (when only preferences are initialized!).

Reply:
Woops, my bad :D